release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1148,6 +1182,8 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 [id="ocp-release-note-insights-operator-bug-fixes_{context}"]
 ==== Insights Operator
 
+Previously, during entitled builds on a Red{nbsp}Hat {product-title} cluster running on IBM-Z hardware, repositories were not enabled. This issue has been resolved. You can now enable repositories during entitled builds on a Red{nbsp}Hat {product-title}  cluster running on IBM-Z hardware. (OCPBUG-32233)
```

Comment:
```suggestion
Previously, during entitled builds on a  Red{nbsp} {product-title} cluster running on {ibm-z-title} hardware, repositories were not enabled. This issue has been resolved. and you can now enable repositories (link:https://issues.redhat.com/browse/OCPBUGS-32233[*OCPBUGS-32233*])
```

Reply:
There are punctuation issues and spacing typos in your suggestion above. 

Adding the attributes in another commit... Thanks.

Reply:
Sorry, about that. I take it that you got the general suggestion that you were missing attributes and didn't format the bug properly.